packages/sui-test/bin/karma/config.js

```diff
@@ -39,7 +39,10 @@ const config = {
                     }
                   ]
                 ],
-                plugins: [require.resolve('babel-plugin-dynamic-import-node')]
+                plugins: [
+                  require.resolve('babel-plugin-dynamic-import-node'),
+                  require.resolve('@babel/plugin-proposal-export-default-from')
```

Comment:
We have this plugin in Babel-preset-SUI

Reply:
The one in the babel-preset-sui is the syntax. (we had the proposal and decided to go only with syntax as is supported by webpack). The problem is karma-webpack doesn't support the same as webpack (they have it fixed but not released yet). For now, jsut, transpile stuff for testing and we will remove this when karma fix it.